hack/bundle-manifests.sh

```diff
@@ -35,7 +35,7 @@ dependencies:
   - type: olm.package
     value:
       packageName: ibm-block-csi-operator
-      version: ">=${BLOCK_CSI_RELEASE:1}"
+      version: ">=${CSI_RELEASE:1}"
```

Comment:
I saw you create a const `CSI_RELEASE_NUMBER` with the same code - can you use it here?